src/main/java/book/store/controller/BookController.java

```diff
@@ -18,40 +22,53 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoint for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "books")
 public class BookController {
     private final BookService bookService;
 
+    @PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
Please doublecheck it works.
As far as I remember hasRole command automatically should prepend value with ROLE_ prefix, so I would expect more
```suggestion
    @PreAuthorize("hasRole('USER')")
```